### ISSUE TITLE
Get rid of kfile.check_space call in load-into-counting

### ISSUE THREAD

**Issue by ctb** [dib-lab/khmer#1167]
load-into-counting doesn't output anything other than a counting graph, so this check is unnecessary.  Also see #1166.

**Comment by bocajnotnef** [comment]
Taking this (and #1166)

**Comment by ctb** [comment]
`git grep check_space\( scripts | cut -d: -f 1` yields the following files which this check should not be in:

```
scripts/abundance-dist-single.py
scripts/count-overlap.py
scripts/load-into-counting.py
scripts/make-initial-stoptags.py
scripts/partition-graph.py
scripts/sample-reads-randomly.py
```

so it should be taken out of all of them, @bocajnotnef  - thx!

**Comment by bocajnotnef** [comment]
Ah, didn't realize that extended beyond load. (as in, that it was equally
unnecessary in those files.)

Will remove.

**Comment by bocajnotnef** [comment]
`load-into-counting` has a call to `check_space_for_hashtable` but not `check_space`--I left the hashtable call in.

**Comment by ctb** [comment]
Fixed in #1170.
